packages/aws-rfdk/lib/deadline/README.md

```diff
@@ -217,6 +217,54 @@ We recommend adding a `script` field in your `package.json` that runs the `stage
 
 With this in place, staging the Deadline Docker recipes can be done simply by running `npm run stage`.
 
+## Thinkbox Docker Images
+
+Thinkbox publishes Docker images for use with RFDK to a public ECR repository. The `ThinkboxDockerImages` construct
+simplifies using these images. To use it, simply create one:
+
+```ts
+// This will provide Docker container images for the latest Deadline release
+const images = new ThinkboxDockerImages(scope, 'Images');
+```
+
+If you desire a specific version of Deadline, you can supply a version with:
+
+```ts
+// Specify a version of Deadline
+const version = new VersionQuery(scope, 'Version', {
+  version: '10.1.11',
```

Comment:
v10.1.11 was pulled, and you're using 10.1.12 elsewhere. Probably a good idea to change that here too.

Reply:
Good catch, changes this to 10.1.12

Reply:
This has been removed. Resolving.